src/Core/Package.vala

```diff
@@ -613,20 +607,21 @@ public class AppCenterCore.Package : Object {
         return cached_search_score;
     }
 
-    private string? name = null;
-    public string? get_name () {
-        if (name != null) {
-            return name;
-        }
+    private string? _name = null;
+    public string name {
```

Comment:
Can we break out this change into another PR? It's a lot of noise in this PR and probably worth doing as a separate commit